docker/api/docker-entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -xe
+
+django-admin collectstatic --noinput
+django-admin migrate --noinput
+exec daphne -b 0.0.0.0 -p $PORT argus.ws.asgi:application
```

Comment:
How will I fill the empty backend with data when using docker-compose? Please document.

Reply:
Should I point to the argus backend docs, or copy&paste the argus backend docs? :-)